chromium_src/chrome/browser/component_updater/widevine_cdm_component_installer.h

```diff
@@ -0,0 +1,50 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_COMPONENT_UPDATER_WIDEVINE_CDM_COMPONENT_INSTALLER_H_
```

Comment:
How about replacing whole `widevine_cdm_component_installer.h` instead of using complex ifdef?
WDYT? @goodov

Reply:
> I simplified the header file in 96734c597cb86a95322f0b1226978fea4abf44c8. I believe we can just use a normal declaration instead of a `#define`. Wdyt @simonhong @goodov?

that should also work, but define approach allows you to keep upstream file included in case something else will pop up there.

Reply:
@goodov I think I was confused by @simonhong's suggestion to replace the whole file; I implemented your suggestion with `#define` now in 0f984d0e98f7744d1dff6da10594c18b8c80ce2c.